raiden_contracts/tests/fixtures/channel.py

```diff
@@ -67,7 +71,7 @@ def get(A: HexAddress, B: HexAddress) -> Tuple:
         channel_identifier = token_network.functions.getChannelIdentifier(A, B).call()
 
         # Test the channel state on chain
-        (channel_settle_timeout, channel_state) = token_network.functions.getChannelInfo(
+        (channel_settle_timeout, channel_state,) = token_network.functions.getChannelInfo(
```

Comment:
Looks like this only returns a two element tuple: https://github.com/raiden-network/raiden-contracts/blob/raiddit/raiden_contracts/data/source/raiden/TokenNetwork.sol#L941

Reply:
This seems to work anyways